licenses.d2iq.yaml

```diff
@@ -6,10 +6,14 @@ ignore:
   - docker.io/bitnami/kubectl:1.26.4
   - docker.io/bitnami/kubectl:1.24.1
   - docker.io/bitnami/memcached:1.6.15-debian-11-r8
+  - docker.io/bitnami/postgresql:11.14.0-debian-10-r28
   - docker.io/bitnami/postgresql:11.16.0-debian-11-r9
   - docker.io/library/busybox:1
+  - docker.io/chrislusf/seaweedfs:3.20
```

Comment:
we might be able to actually pull the licenses for seaweedfs and the fairwinds images below, I'm not totally sure how to figure that out though. @mhrabovcin might know best